lib/form_data.js

```diff
@@ -76,6 +77,19 @@ FormData.prototype.append = function(field, value, options) {
 
   // pass along options.knownLength
   this._trackLength(header, value, options);
+
+  if (!this._entryList[field]) {
+    this._entryList[field] = [];
+  }
+  this._entryList[field].push(value);
+};
+
+FormData.prototype.get = function(field) {
+  return this._entryList[field] ? this._entryList[field][0] : null;
```

Comment:
From the spec

> The get(name) method, when invoked, must return the value of the first entry whose name is `name` from the context object’s entry list, and null otherwise.


Do an `Array.find`